Sources/SwiftProtobuf/StringUtils.swift

```diff
@@ -20,6 +20,41 @@
 
 import Foundation
 
+/*
+  Note: Once our minimum support version is at least Swift 5.3, we
+  should probably recast the following to use
+  String(unsafeUninitializedCapacity:)
+*/
+
+// Note: We're trying to avoid Foundation's String(format:) since that's not
+// universally available.
+
+fileprivate func formatZeroPaddedInt(_ value: Int32, digits: Int) -> String {
```

Comment:
Do we think we'll need this on other integer types later, where it might be good to avoid churn now and make this and the helpers below generic over `BinaryInteger`?

Should it handle negative values, or have a precondition that fails if the value is negative?

Reply:
There's very little of this sort of formatting, so I don't see much need to generalize this.

Precondition is a good idea.  Done.